packages/security/package.json

```diff
@@ -27,12 +27,14 @@
   "homepage": "https://github.com/aerogear/aerogear-js-sdk#readme",
   "devDependencies": {
     "@types/chai": "4.1.7",
+    
     "chai": "4.2.0",
     "mocha": "6.0.2",
     "nyc": "13.3.0",
     "typescript": "3.3.3333"
   },
   "dependencies": {
-    "@aerogear/core": "2.3.0"
+    "@aerogear/core": "2.3.0",
+    "@types/loglevel": "1.5.4"
```

Comment:
Is this included in the other sdk modules?

Reply:
it is use in the auth package ,  straight console.warn will fail the lint